pkg/psmdb/pmm.go

```diff
@@ -50,9 +55,22 @@ func PMMContainer(spec api.PMMSpec, secrets string, customLogin bool) corev1.Con
 			},
 			{
 				Name:  "DB_ARGS",
-				Value: "--uri=mongodb://$(MONGODB_USER):$(MONGODB_PASSWORD)@127.0.0.1:27017/",
+				Value: "--uri=mongodb://$(MONGODB_USER):$(MONGODB_PASSWORD)@127.0.0.1:27017/ --use-profiler",
```

Comment:
is this option needed to PMM2 ?

Reply:
nothing more since RC3
done by @nonemax